src/code/sys_math_atan.c

```diff
@@ -130,10 +130,12 @@ f32 Math_Atan2F(f32 y, f32 x) {
     return Math_Atan2S(y, x) * (M_PI / 0x8000);
 }
 
-s16 Math_FAtan2F(f32 adjacent, f32 opposite) {
-    return Math_Atan2S(opposite, adjacent);
+// Match the OoT implementation of Math_Atan2S
+s16 Math_Atan2S_Depr(f32 x, f32 y) {
```

Comment:
```suggestion
s16 Math_Atan2S_XY(f32 x, f32 y) {
```
Not sure if a consensus was fully reached in Discord, but I at least didn't see anyone outright reject this suggestion.
Same for `Math_Atan2F_Depr`